skyrim-platform/src/platform_se/skyrim_platform/SkyrimPlatform.cpp

```diff
@@ -190,30 +191,36 @@ class CommonExecutionListener : public TickListener
 
     // We will be able to use require()
     JsValue devApi = JsValue::Object();
-    DevApi::Register(devApi, engine,
-                     { { "skyrimPlatform",
-                         [this, engine](JsValue e) {
-                           EncodingApi::Register(e);
-                           LoadGameApi::Register(e);
-                           CameraApi::Register(e);
-                           MpClientPluginApi::Register(e);
-                           HttpClientApi::Register(e);
-                           ConsoleApi::Register(e);
-                           DevApi::Register(e, engine, {}, GetFileDirs());
-                           EventsApi::Register(e);
-                           BrowserApi::Register(e, browserApiState);
-                           Win32Api::Register(e);
-                           FileInfoApi::Register(e);
-                           TextApi::Register(e);
-                           InventoryApi::Register(e);
-                           CallNativeApi::Register(
-                             e, [this] { return nativeCallRequirements; });
-                           e.SetProperty("settings", getSettings, nullptr);
-
-                           return SkyrimPlatformProxy::Attach(
-                             e, [this] { return nativeCallRequirements; });
-                         } } },
-                     GetFileDirs());
+    DevApi::Register(
+      devApi, engine,
+      { { "skyrimPlatform",
+          [this, engine](JsValue e) {
+            auto settings = Settings::GetPlatformSettings();
+
+            EncodingApi::Register(e);
+            LoadGameApi::Register(e);
+            CameraApi::Register(e);
+            MpClientPluginApi::Register(e);
+            HttpClientApi::Register(e);
+            ConsoleApi::Register(e);
+            DevApi::Register(e, engine, {}, GetFileDirs());
+            EventsApi::Register(e);
+
+            if (settings->GetBool("Debug", "ChromiumEnabled", true))
+              BrowserApi::Register(e, browserApiState);
```

Comment:
please move this check to functions of BrowserApi.

In case of chromium disabled throw an exception